ocamltest/ocamltest_config.mli

```diff
@@ -138,7 +128,6 @@ val function_sections : bool
 val instrumented_runtime : bool
 (** Whether the instrumented runtime is available *)
 
-<<<<<<< HEAD
 val naked_pointers : bool
```

Comment:
This will get removed with the ocaml 5 runtime, I think. In any case, upstream removed it.

Reply:
I wasn't sure whether we'd want to keep it around until we've gotten around to deleting the flag -- I'll leave this as a `CR ocaml 5 runtime`.